docs/glossary.md

```diff
@@ -9,5 +9,6 @@ nav_order: 2
 - **Trigger**: A system event caused by the user. Examples: new chat message, PR opened, form submitted, etc.
 - **Action**: A change in the user's state in reaction to a trigger. Examples: advance to next step, approve/reject user’s PR, send a message from bot, etc.
 - **Conditions**: A way to check the user’s state. The state can be the text user entered in the chat, any piece of information attached to the trigger (e.g., PR Status), or information from a 3rd party service (e.g., env var value in K8s)
-- **FlowNode**: when a step starts or triggers are recognized, a Flow will begin. A single element in the flow is called FlowNode. A node is a combination of actions and conditions.
+- **FlowNode**: A step or a trigger that is recognized as a series of actions and conditions.
+- **Flow**: A series of FlowNodes
```

Comment:
Thanks for the suggestions!

Actually the original definition is the correct one. a FlowNode is an element in a larger Flow that is executed when a step starts or when a trigger associated with the step is recognized. We don't really use `Flow` as a term throughout the docs.